src/components/Timer/Timer.module.css

```diff
@@ -18,6 +18,18 @@
    border: 0.07em solid;
    position: absolute;
    left: 5%;
-   bottom: 10%
+   bottom: 10%;
+   animation: pulsate 1s infinite;
 }
 
+@keyframes pulsate {
+   0% {
+      transform: scale(1.0, 1.0); opacity: 0.9;
+   }
+   50% {
+      transform: scale(1.1, 1.1); opacity: 1.0;
+   }
+   100% {
+      transform: scale(1.0, 1.0); opacity: 0.9;
+   }
+}
```

Comment:
This is all stuff from another pull request that still hasn't gone through? Can you try to keep to just one feature on one branch?